assets/admin/students/student-bulk-action-button/index.test.js

```diff
@@ -29,13 +29,20 @@ describe( '<StudentBulkActionButton />', () => {
 			.query( { per_page: 100 } )
 			.reply( 200, courses );
 	} );
+	it( 'Select courses button is disabled by default on render', () => {
+		render( <StudentBulkActionButton /> );
+		const button = screen.getByRole( 'button', {
+			id: 'sensei-bulk-learner-actions-modal-toggle',
```

Comment:
```suggestion
			name: "Select Courses"
```

We should role+ name to get an element, it simulates accessibility requirements.